umm-lib/test/cmr/umm/test/iso_smap/granule.clj

```diff
@@ -66,7 +66,7 @@
       ;; SMAP ISO related-urls does not have title, description or size
       (update-in [:related-urls] related-urls->expected-parsed)
       ;; SMAP ISO does not support orbit-calculated-spatial-domains
-      (dissoc :orbit-calculated-spatial-domains)
+      ;;(dissoc :orbit-calculated-spatial-domains)
```

Comment:
Remove this line.

Reply:
And the comment above since the format does support it now.

Reply:
Done.